tools/restatectl/src/commands/status.rs

```diff
@@ -55,12 +55,27 @@ async fn cluster_status(
     status_opts: &ClusterStatusOpts,
 ) -> anyhow::Result<()> {
     if !status_opts.extra {
-        compact_cluster_status(connection).await?;
+        return match connection.get_nodes_configuration().await {
+            Ok(nodes_config) => compact_cluster_status(nodes_config, connection).await,
+            Err(ConnectionInfoError::MissingMetadata(ident_responses)) => {
+                warn!("Could not read nodes configuration from cluster, using GetIdent responses to render basic list");
 
-        return Ok(());
+                list_nodes_lite(&ident_responses, &ListNodesOpts { extra: false });
+
+                // short-circuit if called as part of `restatectl status`
+                Err(ConnectionInfoError::ClusterNotProvisioned.into())
+            }
+            Err(err) => Err(err.into()),
+        };
     }
 
-    list_nodes(connection, &ListNodesOpts { extra: false }).await?;
```

Comment:
@muhamadazmy was there a reason why we ignore the opts user input here? I reverted that to use opts.extra but please let me know if you disagree.

Reply:
My thinking is that since we now have two different views (compact and extended), if we always use the user input `--extra` to the `listNodes` (hence always true for the extended view) then the nodes table will always have the extra fields which requires one extra call to each node on the table. This was not the case before introducing the compact view. Hence i thought maybe by default the extended (--extra) view shouldn't have these fields by default and only if the user does `nodes list --extra` we can show the extra table fields. 

Thinking about this now, this might not be as intuitive as I hoped, and maybe the `--extra` field can instead be provided multiple times `-e` or `-ee` with each extra `e` we can show more details

Reply:
I also thought maybe it's better if we have 2 different flags to control the verbosity of the output. for example `--full` for the full cluster status view and `--extra` or `--details`, or `--node-details` for what we call now `extra` view.